mappings/net/minecraft/world/level/LevelProperties.mapping

```diff
@@ -64,10 +64,13 @@ CLASS net/minecraft/class_31 net/minecraft/world/level/LevelProperties
 		ARG 2 generatorOptions
 		ARG 3 lifecycle
 	METHOD method_158 updateProperties (Lnet/minecraft/class_5455;Lnet/minecraft/class_2487;Lnet/minecraft/class_2487;)V
+		ARG 1 levelTag
+		ARG 2 playerTag
+		ARG 3 tag
 	METHOD method_185 loadPlayerData ()V
-	METHOD method_29029 readProperties (Lcom/mojang/serialization/Dynamic;Lcom/mojang/datafixers/DataFixer;ILnet/minecraft/class_2487;Lnet/minecraft/class_1940;Lnet/minecraft/class_5315;Lnet/minecraft/class_5285;Lcom/mojang/serialization/Lifecycle;)Lnet/minecraft/class_31;
-		ARG 0 dynamic
-		ARG 1 dataFixer
+	METHOD method_29029 fromDynamic (Lcom/mojang/serialization/Dynamic;Lcom/mojang/datafixers/DataFixer;ILnet/minecraft/class_2487;Lnet/minecraft/class_1940;Lnet/minecraft/class_5315;Lnet/minecraft/class_5285;Lcom/mojang/serialization/Lifecycle;)Lnet/minecraft/class_31;
```

Comment:
Is this pattern of fromDynamic used anywhere else in Yarn? Might be wrong, but have never seen it before.

Reply:
This PR has aged quite a bit, so it may have been in the past but not any more. (this also applies to a few of your other comments)